Kajabi/sage-lib
# [DSS-200]:  Docs site `<head>` refactor

## Description
Currently, the `<head>` sections in our layout files are all identical and contain multiple partials. These changes move head-related elements into 1 partial for improved maintainability.


## Screenshots
No visual changes.


## Testing in `sage-lib`
- Navigate to the [Docs site](http://localhost:4000/pages/index)
- View the source and verify document head appears properly
- Navigate around the docs site and verify pages load properly


## Testing in `kajabi-products`
1. (**LOW**) Refactors Docs site page head to improve maintainability. Documentation only updates, no effect on KP.


## Related
https://kajabi.atlassian.net/browse/DSS-200
